src/main/java/mchorse/mappet/entities/EntityNpc.java

```diff
@@ -580,6 +600,14 @@ public void readEntityFromNBT(NBTTagCompound tag)
         {
             state.id = tag.getString("NpcId");
         }
+
+        if (tag.hasKey("DieOnLoad"))
```

Comment:
It could’ve been just tag.hasKey(“…”) && tag.getBoolean(“…”) plus no space between if and (.

Reply:
Fixed